tools/ir-generator/irclass.cpp

```diff
@@ -251,10 +251,14 @@ void IrDefinitions::generate(std::ostream &t, std::ostream &out, std::ostream &i
 }
 
 void IrClass::generateTreeMacro(std::ostream &out) const {
-    for (auto p = this; p != nodeClass(); p = p->getParent()) out << "  ";
+    {
```

Comment:
Any particular reason for the bracketing? You could just set p in the second loop again, no? Other than that approving early.

Reply:
Also would add brackets to the for loop, I completely missed the `out << " "`; on the first read.

Reply:
No particular reason, just limiting scope of the `p`. But I guess it looks too weird, changed.

Reply:
> Also would add brackets to the for loop, I completely missed the `out << " "`; on the first read.

Yes, this is one of my least favorite features of the code format we use :-(. Personally I tend to use brackets around single line blocks in p4c because of it, even though I never used them in other codebases.

But in general I tend to not change formatting of existing code that I just bug-fix.